src/jvm/main/org/jetbrains/kotlinx/lincheck/verifier/VerifierState.kt

```diff
@@ -27,6 +27,7 @@ package org.jetbrains.kotlinx.lincheck.verifier
  *
  * [VerifierState] lazily counts the test instance state, caches it, and uses in `equals` and `hashCode` methods.
  */
+@Deprecated("Doesn't always improve performance of verification", level = DeprecationLevel.WARNING)
```

Comment:
Please use both Java's and Kotlin's `@Deprecation` annotations.

Reply:
Done